section6Arrays/reverseString/main_test.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import "testing"
+
+func TestReverseString(t *testing.T) {
+	if reverseString("Hello, I am Vatsal.") != ".lastaV ma I ,olleH" {
```

Comment:
use assert statements to test expected and actual values. See examples here: https://github.com/stretchr/testify#assert-package

Here your method to test is: reverseString(input) 
its better you defined the input variable with input
and store the output received in variable named actual. 
like below:
in := "Hello, I am Vatsal."
expected:= ".lastaV ma I ,olleH"

actual := reverseString(in)
assertEquals(t, expected, actual) // look at this methods signature 

if you define in and expected, then you need not to repeat the same logf function.

Reply:
fixed, will use assert package from now on